### ISSUE TITLE
Emojis in right-hand panel heading not rendering.

### ISSUE THREAD

**Issue by RaulThinksStats** [nsgrantham/tidytuesdayrocks#15]
Minor issue, but I noticed that while the emoji renders on the left-hand side of the dashboard, on the right-hand side the emoji's no longer render correctly. Instead, the "careted" unicode for the emoji appears. I suspect it may have to with some update related to how `textOutput` handles the code. Strange though that the emoji would render correctly on the heading below...

I'm running R 3.6.3 on Windows using shiny 1.4.0.2. Let me know if you need additional information to reproduce the error. I've attached a picture for reference after opening up the app on Google Chrome. Thanks for the help and great job on the app 👍. 

![image](https://user-images.githubusercontent.com/40323230/89451048-97490f00-d729-11ea-8f42-247849943be2.png)

**Comment by nsgrantham** [comment]
Hi Raul, thanks for bringing this to my attention. It's interesting that it renders in the dropdown menu, but not on the right-hand panel. My only guess is that you're running Linux and the squirrel/chipmunk emoji isn't natively supported.

In any case, I had to make some changes to the app (performance improvements unrelated to this issue) and I removed the heading in the right-hand panel because it was taking up too much room. So I guess this is fixed now? 😅
